google-cloud-storage/src/main/java/com/google/cloud/storage/UnifiedOpts.java

```diff
@@ -2502,6 +2506,19 @@ private Object readResolve() {
     }
   }
 
+  static final class ObjectContextsFilter extends RpcOptVal<String> implements ObjectListOpt {
```

Comment:
Lets rename this to ObjectFilter as they can be re-used to support other object attributes